mteb/tasks/Retrieval/fra/FQuADRetrieval.py

```diff
@@ -0,0 +1,78 @@
+from __future__ import annotations
+
+import datasets
+
+from mteb.abstasks.TaskMetadata import TaskMetadata
+
+from ....abstasks.AbsTaskRetrieval import AbsTaskRetrieval
+
+
+class FQuADRetrieval(AbsTaskRetrieval):
+    _EVAL_SPLITS = ["test_hasAns", "valid_hasAns"]
+
+    metadata = TaskMetadata(
+        name="FQuADRetrieval",
+        description="This dataset has been built from the French SQuad dataset.",
+        reference="https://huggingface.co/datasets/manu/fquad2_test",
+        dataset={
+            "path": "manu/fquad2_test",
+            "revision": "5384ce827bbc2156d46e6fcba83d75f8e6e1b4a6",
+        },
+        type="Retrieval",
+        category="s2p",
+        eval_splits=_EVAL_SPLITS,
+        eval_langs=["fra-Latn"],
+        main_score="map",
+        date=None,
+        form=None,
+        domains=["Encyclopaedic"],
+        task_subtypes=None,
+        license="apache-2.0",
+        socioeconomic_status=None,
+        annotations_creators=None,
```

Comment:
For the params that currently have `None`, could we please see if we can fill them in based on rough estimates (e.g. date) and `mteb/abstasks/TaskMetadata.py`?

Reply:
Ok done for both